### PR TITLE
test(relay-filter): cleanup

### PR DESCRIPTION
**Pull request by AlejandroCabeza**
# Description
Cleanup relay and filter tests.

# Changes
- [X] Remove legacy relay tests.
- [X] Remove unused imports.
- [X] Add missing payload size test for filter.

# Dependencies
https://github.com/waku-org/nwaku/pull/2101

### CODE REVIEW

File: tests/waku_filter_v2/test_waku_client.nim [hunk 1]
```diff
@@ -1952,164 +2142,8 @@ suite "Waku Filter - End to End":
...
 
         # Then the client receives the message
-        require await pushHandlerFuture.withTimeout(FUTURE_TIMEOUT)
+        check await pushHandlerFuture.withTimeout(FUTURE_TIMEOUT)
         let (pushedMsgPubsubTopic, pushedMsg) = pushHandlerFuture.read()
         check:
           pushedMsgPubsubTopic == pubsubTopic
           pushedMsg == msg
-
-  suite "MessagePushHandler - Msg List":
```
Comment (Ivansete-status) [inline tests/waku_filter_v2/test_waku_client.nim:1961]: Where this suite is being moved?

File: tests/waku_relay/test_waku_relay.nim [hunk 2]
```diff
@@ -1,211 +0,0 @@
-{.used.}
-
-import
-  std/[options, sequtils, strutils],
```
Comment (Ivansete-status) [inline tests/waku_relay/test_waku_relay.nim:4]: Why this file is being removed?

**Review by Ivansete-status — COMMENTED** [review]
Thanks for the PR! It is looking good as tests are expected to wait less time!
Besides, I've added a couple of questions :D

**Review by jm-clius — APPROVED** [review]
LGTM, thanks!

File: tests/waku_relay/test_waku_relay.nim (hunk 2 above)
Comment (AlejandroCabeza) [reply tests/waku_relay/test_waku_relay.nim:4]: Those were legacy tests that have been replaced for the new ones.

File: tests/waku_filter_v2/test_waku_client.nim (hunk 1 above)
Comment (AlejandroCabeza) [reply tests/waku_filter_v2/test_waku_client.nim:1979]: In the beginning I split tests in two to "optimise" the handlers. That is, one set of tests needed a less complex handler, so I created two different handlers, and split the tests between the two. 
After a while, I realised the advantage of having two handlers (a close to zero optimisation), was not worth the simplicity of having just one handler.

**Review by Ivansete-status — APPROVED** [review]
LGTM! :100:

**Pull request merged**